plugins/aws/bedrock/customModelEncryptionEnabled.js

```diff
@@ -0,0 +1,107 @@
+            desiredEncryptionLevelString: settings.bedrock_model_desired_encryption_level || this.settings.bedrock_model_desired_encryption_level.default
+        };
+
+        var desiredEncryptionLevel = helpers.ENCRYPTION_LEVELS.indexOf(config.desiredEncryptionLevelString);
+        var currentEncryptionLevel;
+
+        async.each(regions.bedrock, function(region, rcb){
+            var listCustomModels = helpers.addSource(cache, source,
+                ['bedrock', 'listCustomModels', region]);
+
+            if (!listCustomModels) return rcb();
+
+            if (listCustomModels.err || !listCustomModels.data) {
+                helpers.addResult(results, 3,
+                    `Unable to query for Bedrock custom model list: ${helpers.addError(listCustomModels)}`, region);
+                return rcb();
+            }
+
+            if (!listCustomModels.data.length) {
+                helpers.addResult(results, 0, 'No Bedrock custom model found', region);
+                return rcb();
+            }
+            var listKeys = helpers.addSource(cache, source,
+                ['kms', 'listKeys', region]);
+
+            if (!listKeys || listKeys.err || !listKeys.data) {
+                helpers.addResult(results, 3,
+                    `Unable to list KMS keys: ${helpers.addError(listKeys)}`, region);
+                return rcb();
+            }
+
+            for (let model of listCustomModels.data) {
+                if (!model.modelArn|| !model.modelName) continue;
+               
+                let resource = model.modelArn;
+
+                let getCustomModel = helpers.addSource(cache, source,
+                    ['bedrock', 'getCustomModel', region, model.modelName]);
+
+    
+                if (!getCustomModel || getCustomModel.err || !getCustomModel.data) {
+                    helpers.addResult(results, 3, `Unable to describe Bedrock custom model : ${helpers.addError(getCustomModel)}`, region, resource);
+                    continue;
+                }
+
+                if (getCustomModel.data.modelKmsKeyArn) {
+                    var kmsKeyId = getCustomModel.data.modelKmsKeyArn.split('/')[1] ? getCustomModel.data.modelKmsKeyArn.split('/')[1] : getCustomModel.data.modelKmsKeyArn;
+    
+                    var describeKey = helpers.addSource(cache, source,
+                        ['kms', 'describeKey', region, kmsKeyId]);  
+                    if (!describeKey || describeKey.err || !describeKey.data || !describeKey.data.KeyMetadata) {
+                        helpers.addResult(results, 3,
+                            `Unable to query KMS key: ${helpers.addError(describeKey)}`,
+                            region, getCustomModel.data.modelKmsKeyArn);
+                        continue;
+                    }
+                    currentEncryptionLevel = helpers.getEncryptionLevel(describeKey.data.KeyMetadata, helpers.ENCRYPTION_LEVELS);
+                    
+                } else  currentEncryptionLevel = 2; 
```

Comment:
```suggestion
                } 
```
no need for this else